src/components/tasks/constants.ts

```diff
@@ -1,3 +1,6 @@
 export const INITIAL_TASKS_LIMIT = 3;
 export const ADD_MORE_TASKS_LIMIT = 5;
 export const SEE_MORE = 'See More';
+export const PENDING = 'Pending';
```

Comment:
would suggest using the string in all caps until we have a very specific reason of doing this ?

Reply:
I have updated it now with all caps. Missed to follow this convention.